spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java

```diff
@@ -572,6 +580,40 @@ public void testDeployWithWildcardProperties() throws Exception {
 	}
 
 	@Test
+	public void testDeployWithCommonApplicationProperties() throws Exception {
+		repository.save(new StreamDefinition("myStream", "time | log"));
+		assertThat(appsProperties.getStream().values(), empty());
+		appsProperties.getStream().put("spring.cloud.stream.fake.binder.host","fakeHost");
```

Comment:
On the other hand, the tests already change the state of the repository, registering different stream definitions, so they couldn't normally run in parallel anyway. So probably this could be left like this.